node/src/blockchain/blockchain_interface.rs

```diff
@@ -1076,6 +1070,16 @@ mod tests {
     #[test]
     fn blockchain_interface_non_clandestine_returns_an_error_for_unintelligible_response_when_requesting_token_balance(
     ) {
+        let act = |subject: &BlockchainInterfaceNonClandestine<Http>, wallet: &Wallet| {
+            subject.get_token_balance(wallet)
+        };
+        error_on_requesting_balance(act);
+    }
+
+    fn error_on_requesting_balance<F>(act: F)
```

Comment:
Should start with "assert_..."

Reply:
I'd appreciate if you change it to that.

Well, great work defining this function. 🤙 

Although, I was hoping for more than one occurrence.

I think we can use this design in other places wherever possible.

Reply:
Thanks for pointing me out to the missing usage with another function. :pray: